src/Console/Server.php

```diff
@@ -46,6 +47,8 @@ private function extend(ConsoleApplication $console)
         $app = Application::getInstance();
 
         $events = $app->make(Dispatcher::class);
-        $events->fire(new Configuring($app, $console));
+        $events->fire(new Configuring($app, $console, $dispatcher = new EventDispatcher()));
+
+        $console->setDispatcher($dispatcher);
```

Comment:
I was assuming we would handle events ourselves and pass the exceptions to our array of [reporters registered with the application](https://github.com/flarum/core/blob/12dc4fff573b7cc053e6eaf95756297469cfb7b0/src/Foundation/ErrorServiceProvider.php#L73). That way, the Sentry exception would not need any further changes.

If you would furthermore pipe the error through the registry, [you could use that](https://github.com/flarum/core/blob/12dc4fff573b7cc053e6eaf95756297469cfb7b0/src/Http/Middleware/HandleErrors.php#L61-L67) to determine whether the error is even worth reporting.